actor/builtin/miner/miner_test.go

```diff
@@ -1133,6 +1246,56 @@ func TestMinerGetPoStState(t *testing.T) {
 	})
 }
 
+func TestGetProvingSetCommitments(t *testing.T) {
+	tf.UnitTest(t)
+
+	message := types.NewMessage(address.TestAddress, address.TestAddress2, 0, types.ZeroAttoFIL, "getProvingSetCommitments", nil)
+	var commR1, commR2, commR3 types.CommR
+	copy(commR1[:], th.MakeRandomBytes(32))
+	copy(commR2[:], th.MakeRandomBytes(32))
+	copy(commR3[:], th.MakeRandomBytes(32))
```

Comment:
nit: you could simplify these lines and the `types.Commitments{}` wrap below by calling `th.MakeCommitments()`

Reply:
good call